src/index.ts

```diff
@@ -251,6 +260,21 @@ class LiskMigrator extends Command {
 						configCoreV4 = defaultConfigV4;
 					}
 
+					cli.action.start('Copying genesis block to the Lisk Core executable directory');
+					const liskCoreExecPath = await execAsync('which lisk-core');
```

Comment:
not sure if this is a problem when core two version of core are installed in separate place.. hope it works in ideal scenario

Reply:
This ideally should work because the Migrator does install Lisk Core v4 just before this is being run. But we'll test it out properly.